copy/copy.go

```diff
@@ -109,14 +111,20 @@ func Image(policyContext *signature.PolicyContext, destRef, srcRef types.ImageRe
 		options = &Options{}
 	}
 
-	reportWriter := ioutil.Discard
+	var (
+		hasReportWriter bool
+		writer          io.Writer
+	)
 
 	if options.ReportWriter != nil {
-		reportWriter = options.ReportWriter
+		writer = options.ReportWriter
+		hasReportWriter = true
 	}
-
 	writeReport := func(f string, a ...interface{}) {
```

Comment:
See also https://github.com/containers/image/pull/346/commits/87283c0b56acecab0f749db4fc89e4863a6db52d (“Add `copier.Printf`” from #346).  Not directly applicable, but using a real method, and using a name which is recognized by `go tool vet` would be nice.